notebooks/Notebook3.md

```diff
@@ -2,33 +2,66 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
-last week's work. Answer questions such as:  How, specifically, did the feedback help
-improve the project? Did the feedback point out or offer something you hadn't considered?
-Did it help you make a design decision? Was it helpful in addressing the most pressing
-issues in your project? How will you incorporate the feedback into your work? Will you
-change something about the design, implementation, or evaluation as a result?
+One thing I found especially helpful for my programming this week was suggestions for features.
+For example, some features that were brought up included adding slide numbers, translating elements not off the page, and making it easy to change the order that things show up.
+I added the slide number along with on-screen back and forward controls with the number also functioning as a text input for easy navigation.
+I created parsing for any transformations, so not only translations, but also rotations, scaling, and skewing are now all possible.
+I also kept the syntax clearly separated by transition group (indicated by the `-->`), so it should be easy to reorder by simply cutting and pasting the entire transition in a different place.
+Unfortunately the out-transition is usually on the same key press as the next in-transition, so I may try to distinguish these further in the future.
```

Comment:
I think it's fine that the next slide comes in on the same click that the current slide goes out. Maybe you can make it an option to make it different clicks.